src/ipc_cache.c

```diff
@@ -0,0 +1,204 @@
+/*
+ *
+ * Copyright (C) 2024 Intel Corporation
+ *
+ * Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ *
+ */
+#include "ipc_cache.h"
```

Comment:
please move below to other UMF internal headers

Reply:
Done